judge/ratings.py

```diff
@@ -8,6 +8,45 @@
 from django.utils import timezone
 
 
+BETA2 = 328.33 ** 2
+VAR_INIT = 350. ** 2 * (BETA2 / 212**2)
+VAR_PER_CONTEST = 1219.047619 * (BETA2 / 212**2)
+VAR_LIM = ((VAR_PER_CONTEST ** 2 + 4 * BETA2 * VAR_PER_CONTEST) ** .5 - VAR_PER_CONTEST) / 2
+MEAN_INIT = 1500.
+SD_INIT = VAR_INIT ** .5
+VALID_RANGE = MEAN_INIT - 20 * SD_INIT, MEAN_INIT + 20 * SD_INIT
+TANH_C = 3 ** .5 / math.pi
```

Comment:
Use `sqrt` instead of `** .5` for better readability? See below.

Reply:
Added `sqrt`. I also reordered the constants to improve readability.